monitoring/jaeger-mixin/README.md

```diff
@@ -0,0 +1,82 @@
+# Prometheus monitoring mixin for Jaeger
+
+The Prometheus monitoring mixin for Jaeger provides a starting point for people wanting to monitor Jaeger using Prometheus, Alertmanager, and Grafana. To use it, you'll need [`jsonnet`](https://github.com/google/go-jsonnet) and [`jb` (jsonnet-bundler)](https://github.com/jsonnet-bundler/jsonnet-bundler). They can be installed using `go get`, as follows:
+
+```console
+$ go get github.com/google/go-jsonnet/cmd/jsonnet
+$ go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+```
+
+Your monitoring mixin can then be initialized as follows:
+
+```console
+$ jb init
+$ jb install \
+  github.com/jaegertracing/jaeger/monitoring/jaeger-mixin@master \
+  github.com/grafana/jsonnet-libs/grafana-builder@master \
+  github.com/coreos/kube-prometheus/jsonnet/kube-prometheus@master
+```
+
+In the directory your mixin was initialized, create a new `monitoring-setup.jsonnet` file like the following. Note that this file is yours, any customizations to Prometheus, Grafana, or Alertmanager should take place here.
+
+```jsonnet
```

Comment:
would it make more sense to move this to a sample file?

Reply:
I think @objectiser had a similar question, so, I'm creating a `monitoring-setup.example.jsonnet`.